File: Практическая работа 3/ClientServer/Server/Tasks/Task16.cs
namespace Server.Tasks
{
    internal class Task16
    {
        public static List<double> values()
        {
            var list = new List<double>();
            for (double x = 12; x >= 1; x -= 1.2)
            {
                var z = Math.Tan(x) + 5 * Math.Cos(x - 2);
                list.Add(z);
            }
            return list;
        }
    }
}

File: Практическая работа 2/ClientServer/server/src/main/java/clientserver/server/tasks/Task16.java
package clientserver.server.tasks;

import java.util.List;

public class Task16 {
    public List<Integer> pairsMax(List<List<Integer>> pairs) {
        return pairs.stream().map(
            (List<Integer> l) -> l.stream().max(Integer::compare).get()
        ).toList();
    }
}
